system/cu/cu_main.c

```diff
@@ -291,15 +288,15 @@ int main(int argc, FAR char *argv[])
 
   /* Initialize global data */
 
-  memset(&g_cu, 0, sizeof(struct cu_globals_s));
+  memset(cu, 0, sizeof(struct cu_globals_s));
```

Comment:
Optional: Let's use the same style here and at line 295. So either

```suggestion
  memset(cu, 0, sizeof(*cu));
```
Or change line 295